lms/static/js/certificates/collections/certificate_whitelist.js

```diff
@@ -0,0 +1,58 @@
+// Backbone.js Application Collection: CertificateWhiteList
+/*global define, RequireJS */
+
+;(function(define){
+    'use strict';
+    define([
+            'backbone',
+            'gettext',
+            'js/certificates/models/certificate_exception'
+        ],
+
+        function(Backbone, gettext, CertificateExceptionModel){
+
+            var CertificateWhiteList =  Backbone.Collection.extend({
+                model: CertificateExceptionModel,
+
+                initialize: function(attrs, options){
+                    this.url = options.url;
+                },
+
+                getModel: function(attrs){
+                    var model = this.findWhere({user_name: attrs.user_name});
+                    if(attrs.user_name && model){
```

Comment:
Do you need to check for attrs.user_name here?  It seems that you only need to check for model?